src/main/java/domain/Cards.java

```diff
@@ -0,0 +1,48 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cards {
+    private static final int DECREASE_ACE_VALUE = -10;
+    private List<Card> cards;
+
+    public Cards() {
+        this.cards = new ArrayList<>();
+    }
+
+    public void add(Card card) {
+        cards.add(card);
+    }
+
+    public Card getCard(int index) {
+        return cards.get(index);
+    }
+
+    public int getSum() {
+        Integer sum = cards.stream()
+                .map(s -> s.getCardValue())
+                .collect(Collectors.summingInt(Integer::intValue));
```

Comment:
```suggestion
     int sum = cards.stream()
                .map(Card::getCardValue)
                .mapToInt(Integer::intValue)
                .sum();
```
요런 방식으로도 가능해서 우선 남겨봐요!

Reply:
훨씬 효율적인 것 같습니다! 가독성도 뛰어난 것 같아서 정정하였습니다!